A3A/addons/ultimate/patches/cfgVehicles.hpp

```diff
@@ -261,6 +261,928 @@ class CfgVehicles {
             previewHeight = 30;
         };
     };
+
+    // Custom classes for Antistasi Units / Vehicles in Eden / Zeus
+    class SoldierWB;
+    class I_Soldier_base_F : SoldierWB {
+        class EventHandlers;
+    };
+    class A3U_Rebels_Base: I_Soldier_base_F {
+        scope = 1;
+        scopeCurator = 1;
+        scopeArsenal = 1;
+        faction = "A3U_Rebels";
+        //icon = "";
+        //picture = "";
+        //role = "";
+        //nameSound = "";
+        //textSingular = "";
+        //textPlural = "";
+        uniformClass = "";
+        backpack = "";
+        items[] = {};
+        respawnItems[] = {};
+        linkedItems[] = {};
+        respawnLinkedItems[] = {};
+        weapons[] = {"Throw", "Put"};
+        respawnWeapons[] = {"Throw", "Put"};
+        magazines[] = {};
+        respawnMagazines[] = {};
+        //genericNames = "";
+        //identityTypes[] = {};
+        class EventHandlers {
+            class ADDON {
+                PostInit = "params ['_unit']; [_unit, false] call A3A_fnc_fiaInit;";
+            };
+        };
+    };
+    class A3U_Rebels_Rifleman: A3U_Rebels_Base {
+        scope = 2;
+        scopeCurator = 2;
+        editorSubcategory = "A3U_Militia";
+        displayName = "Rebel Rifleman";
+        //icon = "";
+        //picture = "";
+        role = "Rifleman";
+        nameSound = "veh_infantry_s";
+        textSingular = "infantry";
+        textPlural = "infantry";
+        class EventHandlers : EventHandlers {
+            class ADDON : ADDON {
+                init = "params ['_unit']; _unit setVariable ['unitType', 'loadouts_reb_militia_Rifleman'];";
+            };
+        };
```

Comment:
Replace all `EventHandlers` with this one line:

```suggestion
        GVAR(unitType) = QUOTE(loadouts_reb_militia_Rifleman);
```